pkg/client/client.go

```diff
@@ -856,6 +858,11 @@ func (o *OptimizelyClient) Track(eventKey string, userContext entities.UserConte
 		}
 	}()
 
+	_, span := o.tracer.StartSpan(ctx, "trackHandler", "trackSDK")
+	defer span.End()
+
+	span.SetAttibutes("testTrackKey", "testTrackValue")
```

Comment:
I believe this will be removed. UserContext and eventKey may be interesting, but not sure it'll add any value.

Reply:
Yes, this will be removed in actual implementation. Here the line is just for testing the SetAttributes method with a dummy key pair.